src/library/button/button.tsx

```diff
@@ -25,43 +26,61 @@ export interface IButtonProps {
     leftIcon?: string;
     onPressIn?: any;
     onPressOut?: any;
+    bottomLabel?: string;
 }
 
 export const ButtonComponent = (props: IButtonProps) => (
-    <TouchableOpacity
-        disabled={props.disabled || props.disabledSecondary}
-        onPress={props.onPress}
-        onPressIn={props.onPressIn}
-        onPressOut={props.onPressOut}
-        style={[
-            props.styles.button,
-            props.primary && props.styles.buttonPrimary,
-            props.secondary && props.styles.buttonSecondary,
-            props.disabled && props.styles.buttonDisabled,
-            props.disabledSecondary && props.styles.buttonDisabledSecondary,
-            props.style,
-            {
-                paddingVertical: props?.leftIcon
-                    ? BASE_DIMENSION
-                    : BASE_DIMENSION + BASE_DIMENSION / 2
-            }
-        ]}
-    >
-        {props?.leftIcon && (
-            <Icon name={props.leftIcon} size={normalize(32)} style={props.styles.leftIcon} />
-        )}
-        <Text
+    <View style={[props.styles.container, props.wrapperStyle]}>
```

Comment:
We have to make sure that all `Buttons` used in the App are fixed after this change